bazel/llvm_project/0004_Introduce_filegroups_for_compiler_rt_builtins_runtime.patch

```diff
@@ -0,0 +1,193 @@
+From 53747ea5328ec878fa019961a59ca74eb539d1c1 Mon Sep 17 00:00:00 2001
+From: Chandler Carruth <chandlerc@gmail.com>
+Date: Thu, 24 Apr 2025 05:03:43 +0000
+Subject: [PATCH] Introduce filegroups for compiler-rt builtins runtimes
+
+These filegroups allow downstream projects to package and build
+customized runtime libraries.
+
+The filegroups work hard to use globs and a careful structuring to
+create the structured breakdown of sources needed to target different
+architectures and platforms without having to maintain a complete
+parallel list of sources from CMake.
+---
+ .../compiler-rt/BUILD.bazel                   | 163 +++++++++++++++++-
+ 1 file changed, 160 insertions(+), 3 deletions(-)
+
+diff --git a/utils/bazel/llvm-project-overlay/compiler-rt/BUILD.bazel b/utils/bazel/llvm-project-overlay/compiler-rt/BUILD.bazel
+index 6a6f58995df3..3b930372a54d 100644
+--- a/utils/bazel/llvm-project-overlay/compiler-rt/BUILD.bazel
++++ b/utils/bazel/llvm-project-overlay/compiler-rt/BUILD.bazel
+@@ -122,9 +122,166 @@ cc_library(
+         "lib/fuzzer/Fuzzer*.def",
+     ]),
+     copts = [
+-      # Not using no-sanitize=address per https://github.com/google/sanitizers/wiki/AddressSanitizerContainerOverflow#false-positives
+-      "-fno-sanitize=memory,thread,undefined",
+-      "-fsanitize-coverage=0",
++        # Not using no-sanitize=address per https://github.com/google/sanitizers/wiki/AddressSanitizerContainerOverflow#false-positives
++        "-fno-sanitize=memory,thread,undefined",
++        "-fsanitize-coverage=0",
```

Comment:
Sent you #5354 to try avoiding a patch of the patch

Reply:
Done.